lib/services/livesync/livesync-command-helper.ts

```diff
@@ -15,13 +15,17 @@ export class LiveSyncCommandHelper implements ILiveSyncCommandHelper {
 		return availablePlatforms;
 	}
 
+	/**
+	 * Method sets up configuration, before calling livesync and expects that devices are already discovered.
+	 * @param devices List of discovered devices
```

Comment:
you have to define the type of the parameter:
https://github.com/Microsoft/TypeScript/wiki/JsDoc-support-in-JavaScript